gpu/segmentation/src/extract_clusters.cpp

```diff
@@ -37,12 +37,26 @@
 
 #include <pcl/impl/instantiate.hpp>
 #include <pcl/point_types.h>
-//#include <pcl/gpu/segmentation/gpu_extract_labeled_clusters.h>
-#include <pcl/gpu/segmentation/impl/gpu_extract_clusters.hpp>
```

Comment:
I think clang tidy would reorder these includes back, so might as well restore original order?

Reply:
Hm, I can't restore the original lines then the code does not compile. I however did clang-format these lines. Your hypothesis about the order turned out to be correct.